lmdeploy/turbomind/deploy/config.py

```diff
@@ -53,13 +53,16 @@ class ModelConfig:
 class AttentionConfig:
     rotary_embedding: int = 128
     rope_theta: float = 10000.0
+    attention_factor: float = None
```

Comment:
https://github.com/huggingface/transformers/blob/f2c388e3f946862f657acc1e21b272ec946fc66c/src/transformers/modeling_rope_utils.py#L189
There is another parameter "partial_rotary_factor"

Reply:
It seems the `partial_rotary_factor` changes the dim of `default`, `dynamic ntk` and `yarn`